src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.spec.ts

```diff
@@ -363,4 +367,108 @@ xdescribe('CreateEditNewsComponent', () => {
     component.removeFilters(notActiveFilter);
     expect(component.tags().length).toBe(0);
   });
+
+  it('should create form with 5 controls', () => {
+    expect(component.form.contains('title')).toBeTruthy();
+    expect(component.form.contains('source')).toBeTruthy();
+    expect(component.form.contains('content')).toBeTruthy();
+    expect(component.form.contains('tags')).toBeTruthy();
+    expect(component.form.contains('image')).toBeTruthy();
+  });
+
+  it('should update the value of the title, content, source fields', () => {
+    const title = component.form.controls.title;
+    const source = component.form.controls.source;
+    const content = component.form.controls.content;
+
+    title.setValue('Title');
+    source.setValue('Source');
+    content.setValue('content');
+    expect(title.value).toBe('Title');
+    expect(source.value).toBe('Source');
+    expect(content.value).toBe('content');
+  });
+
+  it('should test form validating', () => {
+    const form = component.form;
+    const contentInput = form.controls.content;
+    const titleInput = form.controls.title;
+    const invalidString = `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab dicta doloremque illum libero
+     nihil recusandae tempora veniam? Ad architecto aspernatur consectetur consequuntur culpa cupiditate dignissimos
+     dolorem ea earum enim error eum inventore ipsam ipsum iure laborum laudantium maxime modi molestiae natus nobis
+     numquam odit optio quae quaerat quas quidem rem repellat repellendus repudiandae, sequi sint sit ullam vel vero
+     voluptate voluptatem! Ab, cum et harum quas repellendus saepe voluptatem. Distinctio dolorem molestiae mollitia
```

Comment:
why do we have 75 files in this PR?

Reply:
@JimHolman1199 oliachipko
Merge branch 'dev' into base-feature/#2795-ubs-admin-cabinet
we have these files as the result of merging develop into our branch to have our branch up-to-date with develop...